src/components/LandingPage/LandingPage.js

```diff
@@ -368,7 +345,7 @@ const LandingPage = () => {
     return (
         <LandingPageWrapper>
             <BannerSection>
-                <TagLine>We decentralize DeFi</TagLine>
+                <TagLine>WE PUT DECENTRALISATION BACK INTO DEFI</TagLine>
```

Comment:
@pimato The "Back" sounds weird to me here.  Should we remove that?

Reply:
yes lets do it!